cmd/server-mux.go

```diff
@@ -469,6 +474,13 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 				return
 			}
 
+			// Return ServiceUnavailable when we hit a limit of number
```

Comment:
This is better --> `// Return HTTP service unavailable when we have reached maximum concurrent HTTP requests.`